<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>NextClippedPolygon</class>
 <widget class="QDialog" name="NextClippedPolygon" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Next Clipped Polygon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="widget" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="fishery_text" >
          <property name="minimumSize" >
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Current Fishery :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="fishery_line" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_4" >
     <property name="margin" >
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="shapeNum_label" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Shape #</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>88</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_1" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Pennies:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pl_label" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="widget_2" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbnNextShape" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Next Shape</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
